bots/psuteam7bot/castle.js

```diff
@@ -264,6 +268,13 @@ castle.findPosition = (self) => {
             return 1;
         }
     });
+    self.teamCastles.sort((a,b) => {
+        if(movement.getDistance(self.me, a) > movement.getDistance(self.me, b)) {
```

Comment:
I think this okay, but I vaguely recall that it was better to sort by id (to ensure that `self.teamCastles[0]` is yourself). Can't remember for sure tho